config/sampler/conf/gflownet.yaml

```diff
@@ -130,7 +130,7 @@ logger:
   _target_: gflownet.utils.logger.Logger
 
   do:
-    online: False # <---
+    online: True # <---
```

Comment:
I would keep this `False`. Otherwise debugging runs, failed runs will generate wandb runs which end up creating a lot of clutter.